### ISSUE TITLE
Images are not showing

### ISSUE THREAD

**Issue by gallart** [servergrove/KnowledgeBase#20]
For a "weird" reason, images are not showing on the articles.

![capture decran 2014-06-10 a 10 57 27](https://cloud.githubusercontent.com/assets/2728195/3227616/36cc24d0-f07d-11e3-973a-0615565f88b7.png)

Any idea?

**Comment by pgodel** [comment]
Not really sure what is going on. It appears the Markdown is not being properly parsed. Does other markdown syntax work?

**Comment by gallart** [comment]
Is there any PHP Requirement for markdown parsing?

**Comment by pgodel** [comment]
No, the markdown is currently done with https://github.com/chjj/marked
